page.codeberg.JakobDev.jdMacroPlayer.yaml

```diff
@@ -0,0 +1,73 @@
+id: page.codeberg.JakobDev.jdMacroPlayer
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: jdmacroplayer
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_PYWEBENGINE=1
+finish-args:
+  - --talk-name=org.freedesktop.Flatpak
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - name: python3-desktop-entry-lib
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "jeepney" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/ae/72/2a1e2290f1ab1e06f71f3d0f1646c9e4634e70e1d37491535e19266e8dc9/jeepney-0.8.0-py3-none-any.whl
+        sha256: c0a454ad016ca575060802ee4d590dd912e35c122fa04e70306de3d076cce755
+        x-checker-data:
+          name: jeepney
+          packagetype: bdist_wheel
+          type: pypi
+
+  - name: ydotool
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        archive-type: tar-gzip
+        url: https://api.github.com/repos/ReimuNotMoe/ydotool/tarball/v1.0.4
+        sha256: 6c6ff6d8ca997a514f86efef803ef71b2dd139eea5eba8490e8613424fee49bd
+        x-checker-data:
+            type: json
+            url: https://api.github.com/repos/ReimuNotMoe/ydotool/releases/latest
+            version-query: .tag_name
+            url-query: .tarball_url
+    modules:
+      - name: scdoc
+        buildsystem: simple
+        build-commands:
+          - make PREFIX=${FLATPAK_DEST}
+          - make PREFIX=${FLATPAK_DEST} install
+        sources:
+          - type: archive
+            url: https://git.sr.ht/~sircmpwn/scdoc/archive/1.11.3.tar.gz
+            sha256: 4c5c6136540384e5455b250f768e7ca11b03fdba1a8efc2341ee0f1111e57612
+        cleanup:
+          - "*"
+
+  - name: jdMacroPlayer
+    buildsystem: simple
+    build-commands:
+      - pip install --no-deps --no-build-isolation --prefix=$FLATPAK_DEST .
+      - python ./install-unix-datafiles.py --prefix=$FLATPAK_DEST
```

Comment:
metainfo is commented for some reason https://codeberg.org/JakobDev/jdMacroPlayer/src/branch/main/install-unix-datafiles.py#L57